### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by Kasia9311**
Adding new function

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @Kasia9311

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Kasia9311 for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,21 @@
...
+        	}else if (command == "divide"){
+        		if (second == 0){
+       	 		return "Division by 0";
+       	 	}else{
+       	 		return std::to_string(first / second);
+       	 		}
+        	}	
+    else{	
+        return "Invalid data";
+    }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:21]: Please be consistent with spaces.

Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else, {, }, etc...

Also wrong indentation

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,21 @@
...
     return "";
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:22]: Dead code. Remove it.

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -3,5 +3,21 @@
...
+    if (command == "add"){
+        return std::to_string(first + second);
+    }else if (command == "subtract"){
+        return std::to_string(first - second);
+        } else if (command == "multiply"){
+        	return std::to_string(first * second);
+        	}else if (command == "divide"){
+        		if (second == 0){
+       	 		return "Division by 0";
+       	 	}else{
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:15]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**